src/main/java/it/smartcommunitylab/aac/oidc/provider/OIDCAuthenticationProvider.java

```diff
@@ -253,14 +253,13 @@ public Authentication doAuthenticate(Authentication authentication) throws Authe
 
             return auth;
         } catch (OAuth2AuthenticationException e) {
-            throw new OIDCAuthenticationException(
+            logger.debug(
+                "exception occurred when authenticating with OIDC provider {}, error information {}, {}",
+                getProvider(),
                 e.getError(),
-                e.getMessage(),
-                authorizationRequestUri,
-                authorizationResponseUri,
-                null,
-                null
+                e.getMessage()
             );
+            return null;
```

Comment:
the previous behavior was throwin an exception, here you swallow the ex and return null. This is a major change. Please revert. Just add logging

Reply:
The removal of the thrown exception was not originally planned and was added due to a misunderstanding of comment
https://github.com/scc-digitalhub/AAC/pull/479#discussion_r1462202808

upon second thought, did you intend to drop the second argument (i.e. the throwable object `e`) from the `loggin.debug(... , e)` method?

Reply:
yes, I meant remove the throwable "from logging", sorry if it wasn't clear enough. Removing the exception thrown from the method is a whole different story, and will break the login flow.